lesson19_task3.py

```diff
@@ -0,0 +1,32 @@
+"""
+Create your own implementation of an iterable, which could be used inside for-in loop.
+Also, add logic for retrieving elements using square brackets syntax.
+"""
+
+
+class MyIterable:
```

Comment:
Nice implementation, once again, only type hints could be added.
But your codestyle is already pretty much fine, I would expect that level as of a professional developer.